### PR TITLE
UIOR-237 base structure for order templates settings

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIOR-237

## Approach

routing for order templates views and layer usage for editor to have full width 

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/OrderTemplates/OrderTemplates.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <OrderTemplatesList
+            label={label}
+            rootPath={path}
+          />
+        )}
+      />
+      <Route
+        exact
+        path={`${path}/create`}
+        render={() => <OrderTemplatesEditor />}
```
Comment (aliaksei-chumakou) [inline src/settings/OrderTemplates/OrderTemplates.js]: no props passed? is it legal?

Comment (NikitaSedyx) [reply src/settings/OrderTemplates/OrderTemplates.js]: they will be in future

File: src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class OrderTemplatesEditor extends Component {
+  render() {
+    return (
+      <Layer isOpen>
+        <Pane
+          id="order-settings-order-templates-editor"
+          defaultWidth="fill"
+          paneTitle="Order template editor"
+          dismissible
```
Comment (aliaksei-chumakou) [inline src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js]: dismissible without onClose? does it work well?

Comment (NikitaSedyx) [reply src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js]: no, it doesn't as it's just set up, onClose is required in this case

File: src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+} from '@folio/stripes/components';
+
+class OrderTemplatesEditor extends Component {
+  render() {
+    return (
+      <Layer isOpen>
+        <Pane
+          id="order-settings-order-templates-editor"
+          defaultWidth="fill"
+          paneTitle="Order template editor"
```
Comment (AndreiShumski2) [inline src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js]: Should it be translated?

Comment (NikitaSedyx) [reply src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js]: yes, but not now

**Pull request merged**